packages/react-stylesheets/src/StylesheetContext.tsx

```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 export type HTMLDirection = 'rtl' | 'ltr' | 'auto';
 
+const defaultDocument = { document: 'document' };
```

Comment:
Hmm; providing a default mock document would break the renderStyles default method; would it be better to just ignore doing work when the `target` is unavailable?